web/src/components/faq.js

```diff
@@ -97,17 +98,12 @@ const FAQ = ({ className }) => {
   // See https://github.com/tmrowco/electricitymap-contrib/issues/2161.
   const ref = useRef(null);
   setTimeout(() => {
-    if (ref && ref.current) {
-      const links = ref.current.querySelectorAll('.entry-link');
-      // This seems to be the most browser-compatible way to iterate through a list of nodes.
-      // See: https://developer.mozilla.org/en-US/docs/Web/API/NodeList#Example.
-      Array.prototype.forEach.call(links, (link) => {
-        const entryKey = link.href.split('#')[1];
-        if (entryKey) {
-          link.onclick = () => { setAnswerVisible(entryKey, true); };
-        }
-      });
-    }
+    modifyDOM(ref, '.entry-link', (link) => {
```

Comment:
Maybe worth adding a TODO that states this should be done using react?

Reply:
There's already a TODO comment a few lines above, I expect this will be easier to do via hash state once we add React Router.